app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,29 @@
+class Api::V1::CommentsController < ApplicationController
+  before_action :find_post
+
+  def index
+    # http://127.0.0.1:3000/api/v1/users/1/posts/102/comments
+    @comments = @post.comments
+    render json: @comments
+  end
+
+  # POST http://127.0.0.1:3000/api/v1/users/1/posts/102/comments
+  def create
+    @user = current_user
+    @comment = @post.comments.build(comment_params.merge(author: @user))
+
+    if @comment.save
+      render json: @comment, status: :created
+    else
+      render json: { errors: @comment.errors.full_messages }, status: :unprocessable_entity
+    end
+  end
```

Comment:
- [ Required ] I think it would be nice if this API `endpoint` works as expected, currently, when I try to create a comment from the `Insomnia` REST client I get the error below. I think this is because of the reason below.
   
   - [ ] In `application_controller.rb` on line `2`  _protect_from_forgery with: :exception_ prevents your app from recieving `POST` requests from anything other than a form. So if I send a `POST` request with a `JSON` body that's why I get an error. I would like to suggest you solve this by replacing the code on line `2` with the code below. Please make sure that it's possible to add a `comment` to a `post` using the `http://127.0.0.1:3000/api/v1/users/1/posts/1/comments` endpoint.

```ruby
  protect_from_forgery with: :null_session
```  

## Error when adding comments to a post
![Screenshot from 2023-09-15 23-01-01](https://github.com/ProgramKingAbel/tubongeApp/assets/16392046/b1bc5b9d-4453-42b4-87ae-503762daba4a)